client/src/components/User/UserBeaconSettings.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+  <b-row class="ml-3 mb-1">
+    <i class="pref-icon pt-1 fa fa-lg fa-broadcast-tower"/>
+    <div class="pref-content pr-1">
+      <a id="beacon-settings" href="javascript:void(0)"
+      ><b v-b-modal.modal-beacon v-localize>Manage Beacon</b></a
+      >
+      <div v-localize class="form-text text-muted">Contribute variants to Beacon</div>
+      <b-modal
+          size="xl"
+          id="modal-beacon"
+          ref="modal"
+          ok-only
+          title="Manage Beacon"
+          title-tag="h1"
+          @show="loadSettings">
+
+        <!-- Explanation text-->
+        <p>
+          This setting will enable the possibility to share your Variants (in VCF (vcf) or compressed VCF (vcf_gz))
+          with the entire community in a secure way without sharing the entire file or Galaxy history. This is possible
+          by utilizing the <a href="https://beacon-project.io">Global Alliance for Genomics & Health Beacon Project</a>.
+          <br>
+          <br>
+          We turn Galaxy into a Beacon server and offer your variants anonymously via the Beacon Network. All shared
+          Variants will be merged to a single dataset. If someone searches in the Beacon Network for a specific variant
+          that is in one of your datasets our Galaxy server will return
+          <span class="cursive">“Yes, we have seen such a variant”</span>. Nothing more.
+          <br>
+          <br>
+          The user has the possibility to contact the Admin of the Galaxy server and we will in turn contact
+          you and ask if you want to contact this user and negotiate a potential data access.
+          Sharing is essential in Galaxy, secure sharing of sensitive data is now possible using Beacon.
```

Comment:
Here's my suggestion for rephrasing this:
```
The <a href="https://beacon-project.io">Global Alliance for Genomics & Health Beacon Project</a> enables safe sharing of human genetic variants.

Galaxy lets you use the Beacon protocol to share genetic data directly from your analysis with the scientific community in the following anonymous way:

For participating users, we will merge variant lists to be shared into a single Beacon dataset and make that dataset accessible through a Beacon server. If someone queries the server for a specific variant that is in our Beacon dataset, the server will reply with <span class="cursive">“Yes, we have seen such a variant”</span>. Nothing more.

The user that issued the query then has the possibility to contact a Galaxy server admin who can link the variant call in question to particular Galaxy users. If you are among the users that shared the variant, the admin will, in turn, contact you and ask if you want to contact the user that initiated the query to negotiate further information exchange or data access.
```

Reply:
Very good suggestion :)